build.gradle

```diff
@@ -41,6 +41,7 @@ sourceCompatibility = 11
 targetCompatibility = 11
 
 repositories {
+    jcenter()
```

Comment:
remove this - this is deprecated. Use jitpack.

Reply:
this is necessary for now. it complains about not finding some package when running this locally.